src/components/Type.jsx

```diff
@@ -36,6 +36,6 @@ const types = {
 
 export default function Type({ children }) {
     const type = types[children]
-    return <span class={`type type-${type.class ?? "obj"}`}><a href={type.link}>{children}</a></span>
+    return <span className={`type type-${type.class ?? "obj"}`}><a href={type.link}>{children}</a></span>
```

Comment:
Why is this necessary?

Reply:
I made that change to get rid of an error (you can't use `class` in React), but I removed the change now.

Reply:
Huh, I am not aware of that error. Please re-add it if you like.

Reply:
The `Parameter` component now has some warnings about `class` and a missing `key` prop. I could fix them here, but it might look weird in the final diff because this PR will be squashed.